### DISCUSSION TITLE
Supports use wildcards on "Allowed hosts"  configure option

### DISCUSSION THREAD

**Discussion by czd890** [grafana/grafana-infinity-datasource#795] [Ideas]
![image](https://github.com/grafana/grafana-infinity-datasource/assets/4011549/d2f4b77f-908b-4185-abb8-a55c3a68f6e1)

**Comment by a-abella** [comment]
I would like this also. I am using infinity to unify/flatten active alerts from many dozens of alertmanagers and grafana-built-in alertmanagers so all alerts can be visualized in one place. For us that means adding `https://*-alertmanager.company.tld` x20, and `https://*-grafana.company.tld` x20.

Wildcard would not only be nicer ux but would allow us to expand without forgetting to update the allowed hosts list.

**Comment by tobiasehlert** [comment]
This would also be very handy in cases, where you have very long URLs. For instance hosted environments like run on `*.azurewebsites.net` which quickly become very long.
